include/scxml_core/scxml_sm_interface.h

```diff
@@ -23,11 +25,24 @@ class ScxmlSMInterface
 public:
   ScxmlSMInterface(const std::string& scxml_file);
 
+  /**
+   * @brief checks if an event exists
+   */
+  bool eventExists(const QString& event, std::set<std::pair<QString, QString>> events);
+
+  /**
+   * @brief gets the state to which a desired transition occurs
+   * @param search_text - insert transition text you'd like to match
+   * @throws if you don't have that transition, it will return itself as it's neighbor
+   */
+
+  const QString getNeighbor(const QString& state, const QString& search_text);
```

Comment:
Rename the argument from `search_text` to `transition` which is more meaningful

```suggestion
  /**
   * @brief Returns the state to which a desired transition leads
   */
  const QString getNeighbor(const QString& state, const QString& transition) const;
```

Reply:
Still a relevant comment